### PR TITLE
AirportTransportSystem 

### PR DESCRIPTION
**Pull request by Ibraga96**
Added Complex Model for AirportTransportSystem , I was using the ControllerTemplate for model controller , can't do it by myself because still can't understand how to configure data sharing between two tables automatically, now it works nice with default data from Initializer , but some problems appear with GUID when we want to create new Order with new Passenger, also Added Buttons, Enums , Initializer .ToDo: Add new model ForCompanies

### CODE REVIEW

File: UtopiaCity/Models/Airport/TransportManagerSystem/Enums/TypeOfTransportForPassengers.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace UtopiaCity.Models.Airport.TransportManagerSystem.Enums
+{
+    public enum TypeOfTransportForPassengers
+    {
+        Car = 0,
+        BusNumber4 = 1,
```
Comment (crocomoth) [inline UtopiaCity/Models/Airport/TransportManagerSystem/Enums/TypeOfTransportForPassengers.cs:11]: I would say that busnumber should not be a enum
Bus can be - then we can specify number in some field

File: UtopiaCity/Models/Airport/TransportManagerSystem/ForPassenger.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace UtopiaCity.Models.Airport.TransportManagerSystem
+{
+    public class ForPassenger
+    {
+        [ScaffoldColumn(false)]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        [Key]
```
Comment (crocomoth) [inline UtopiaCity/Models/Airport/TransportManagerSystem/ForPassenger.cs:10]: It should be key by default as well

**Review by crocomoth — APPROVED** [review]

File: UtopiaCity/Models/Airport/TransportManagerSystem/Enums/TypeOfTransportForPassengers.cs (hunk 1 above)
Comment (Ibraga96) [reply UtopiaCity/Models/Airport/TransportManagerSystem/Enums/TypeOfTransportForPassengers.cs:11]: ok i will change

File: UtopiaCity/Models/Airport/TransportManagerSystem/ForPassenger.cs (hunk 2 above)
Comment (Ibraga96) [reply UtopiaCity/Models/Airport/TransportManagerSystem/ForPassenger.cs:10]: and this one

**Review by IvanSultanov — APPROVED** [review]

**Pull request merged**